### ISSUE TITLE
Make "Buy Print Version" button replacable by an  "Open Access Download E-Book" button

### ISSUE THREAD

**Issue by Cutuchiqueno** [ManifoldScholar/manifold#1214]
## Feature Description

For Open Access online only publications, not distributed somewhere else, it would be great if the "Buy Print Version ___$" Button could optionally be replaced with a "Download E-Book" button.

## *Why* is this feature important? *Who* does it help?

There are scenarios (such as in our case) where Manifold will be used as the main and primary publishing platform (without any associated publisher or publishing location). In such cases a workaround such as "Buy Print Version 0$" (or Euros) is really only a workaround and does not give a nice impression considering the "symbolic capital" the platform should give to the publication within an academic environment.

## User Stories

An institution or faculty might want to use Manifold Scholarship in order to publish Open Access online-only publications such as conference proceedings for which no use of other publishing platforms or distribution channels is planned.

## Design Notes

Making the Buttontext of "Buy Print Version" configurable and the segment for the amount and the currency optional could be an approach

Another one would be to offer a set of options such as

1. Buy Print Version  ___$
2. Download E-Book Open Acces (+ license options)
3. …

## Development Notes

I do not know how the whole backend works and thus can not tell what is necessary to implement this.

**Comment by mkgold** [comment]
Thanks so much for submitting this @Cutuchiqueno. I'd like to see that button be fully configurable, with both the button text and the URL being fully configurable. For an OER content type, for instance, I could see someone wanting to use that button to link to, say, a syllabus or course homepage

**Comment by zdavis** [comment]
Related to #1209. I see us addressing this as part of that unit of work, wherein we make the project detail hero _significantly_ more flexible. That said, if this is a deal-breaker for @Cutuchiqueno or someone else using Manifold in the interim, I'm open to coming up with relatively simple interim solutions.

**Comment by jeffpooley** [comment]
I want to echo the request — at mediastudies.press we are planning to use Manifold as the main publishing site, with no companion print books. So I would be thrilled with any stopgap solution.

**Comment by mkgold** [comment]
That's fantastic news @jeffpooley !!!

**Comment by zdavis** [comment]
@susanmpls mentioned in Slack the other day that some users would like the option of both a download button _and_ a purchase button. 

Right now, we're planning on including this in the next Manifold release. Tagging issue for 2.0 Milestone and removing the "under discussion" label as I think this is actionable as is. @laelito, do you want to weigh in on design here? We're taking about adding a download button in the project hero area:

![image](https://user-images.githubusercontent.com/255909/43591509-6114fdc0-9628-11e8-8dc5-8a36f55255f9.png)

Do you think it would be sufficient to just render the download button under the purchase button, in the same style?

**Comment by laelito** [comment]
Hi all, take a look at these two design options, and please weigh-in with preferences or suggestions.

Option 1: This is a simple duplication of the "buy" button styling. This option works well to call attention to the download link if there is no print version available. We are leaning towards option 1 for this reason.

Option 2: However, if publishers would prefer to emphasize the availability of the print version we could use a simple text label style button.

![download_button_options](https://user-images.githubusercontent.com/26985828/43602187-1318af6a-9644-11e8-8646-cb44eca9446e.jpg)

**Comment by susanmpls** [comment]
Lael, these are great and both of these options would work well, and I have a very strong preference for door #2. 
 
The rule around "download" in Option 1 means it is a giant button that feels heavy on the page, the overall page seems weighted towards the bottom right corner. It's too prominent a feature because it draws the eye away from the start reading button. Blerg.

In Option 2 the "download digital version" makes it clear you can download, but it doesn't give it prominence over reading on Manifold. It's easy to spot for those seeking a download but it doesn't compete with the "start reading" button. I would rephrase to "Download Ebook." (Because Manifold is a digital verison, too, just not a downloadable one.)

**Comment by zdavis** [comment]
I leaned toward #1 initially, but I'm feeling pretty swayed by Susan's reasoning here, so put my vote on #2 as well.

**Comment by jeffpooley** [comment]
Thanks for the mock-up and the openness to feedback! I agree about option 2, but hope that there is a setting—if that’s the solution the team goes with—to remove the BUY PRINT VERSION button altogether. I saw interest expressed earlier, and it’s my own too, to use the amazing Manifold software (thanks for your incredible work!!!) for fully open-access, digital-only works. In that scenario, of course, there could be an option to hide the “BUY PRINT VERSION” button altogether. Or at least an option to change its language, etc.

**Comment by susanmpls** [comment]
Agree with Jeff, about removing buy print, especially if publishers are only publishing digitally. 

Another question: Is the download ebook function something that could lead to a shopping cart instead of a free download? (Asking for all the publishers that will ask me if that's possible.)

**Comment by zdavis** [comment]
Both of these buttons would only appear if the project were configured to support them. I envision a few fields that you can set on the project record (the purchase ones already exist):

Purchase Price
Purchase URL
Downloadable Ebook (attachment)
Downloadable Ebook URL (if stored externally, could be a link to an epub purchase location)

If the purchase info is entered, Manifold will render the buy link. If an epub is attached or if an external URL is entered, Manifold will render the download link. So it will be up to the publisher to decide which (or both) of these to use. Sound OK?

ZD

**Comment by jeffpooley** [comment]
Sounds great! Thanks!!

**Comment by susanmpls** [comment]
yes. perfect. thank you!

**Comment by zdavis** [comment]
When we get to #1209, this will become configurable to the point where the publisher can choose the language in the button. There are lots of variations here—buy the print version, buy the digital version, buy both, download the free version, etc. We need to give publishers the ability to choose the language on these call to actions instead of making assumptions, which is what I hope we're able to do in #1209.

**Comment by susanmpls** [comment]
T, Sounds like that will be resolved via #1209 -S

**Comment by zdavis** [comment]
Yeah, and once we get to the paywall part, it will definitely affect this further... but that's in the more distant future for now.

**Comment by Cutuchiqueno** [comment]
as mentioned in my initial post to this issue we will probably not offer any buying options. This said, I am very much in favour of Option 1. Not only do I not understand why a download only option should be treated differently aesthetically, I also like the idea of a "Download Open Access E-Book" message (or something similar) being prominent. It is not just a Download Link but also a message, a message that deserves to be shown as prominent as the the scenario of comercial distribution.

**Comment by Cutuchiqueno** [comment]
@zdavis Thank you for the explanation and the work on it. Looking forward to it.

**Comment by zdavis** [comment]
Yay, this is done and will be in the 2.0 release (release candidate likely coming next week). We ended up in a compromise position—if the only option is to download a book, the download link will be given the more prominent button treatment. If it is possible to buy and download the book, the buy button gets the more prominent treatment and the download button gets the visually less prominent treatment.

Thanks everyone for your feedback on this! We really appreciate your engagement!
